files/image_config/interfaces/interfaces.j2

```diff
@@ -22,6 +22,9 @@ iface mgmt
 # The loopback network interface
 auto lo
 iface lo inet loopback
+   address 127.0.0.1
+   netmask 255.255.0.0
```

Comment:
How about line 16 above?

Reply:
Thanks for pointing that out Prince.
This change doesn't seem mandatory for what we're trying to achieve but I changed it here too for consistency purposes.
This way all loopback use 127.0.0.1/16.